scripts/install.py

```diff
@@ -137,22 +144,29 @@ def main():
         install_gc3pie_from_pypi(options.target, options.features)
 
     print("""
-===============================
-Installation of GC3Pie is done!
-===============================
-
-In order to work with GC3Pie you have to enable the virtual
-environment with the command:
-
-    . {target}/bin/activate
-
-You need to run the above command on every new shell you open before
-using GC3Pie commands, but just once per session.
+            ===============================
+            Installation of GC3Pie is done!
+            ===============================
+
+            In order to work with GC3Pie you have to enable the virtual
+            environment with the command:
+    """)
+    if not default.ANACONDA:
+        print("""
+            . {target}/bin/activate
+        """.format(target=options.target))
+    else:
+        print("""
+            source activate {target}
+        """.format(target=os.path.basename(os.path.normpath(options.target))))
```

Comment:
I'd rather define `target_name = os.path.basename(...)` than repeating the same expression over and over.